### PR TITLE
Website update

### PR DESCRIPTION
**Pull request by okawo80085**
Added Nick to the TEAM page, updated the text in the HOME page a bit and updated the logo.

Signed-off-by: Oleg Vorobiov <oleg.vorobiov@hobovrlabs.org>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,18 +65,18 @@ <h2>What we do</h2>
         <h2>Getting started</h2>
         <h4>Using the Driver</h4>
         <p>
-            Build a tracker, a controller, an HMD, or anything else your heart desires, either using one of our guides or by coming up with something new. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
+            Build a tracker, a controller, an HMD, or anything else your heart desires, either using whatever hardware your heart desires. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
```
Comment (SimLeek) [inline index.html]: * either needs two things, but this is just one thing 'whatever your heart desires'
* 'heart desires' is written twice, which looks redundant

File: index.html [hunk 2]
```diff
@@ -73,18 +65,18 @@ <h2>What we do</h2>
...
         </p>
 
         <h4>Contributing</h4>
         <p>
-            The best way to get in touch with us is through our <a href="https://github.com/HoboVR-Labs">GitHub organization</a>.
-            <br>We'll be happy to review your pull requests or patches.
+            A good place to start is our <a href="https://github.com/HoboVR-Labs/issue_tracker/issues">issue tracker</a>, there you can find feature requests, bug reports, archives, conventions and other issues. It's a good place to start.
```
Comment (SimLeek) [inline index.html]: Said 'good place to start' at both the start and end of this paragraph, which feels redundant.

File: index.html (hunk 1 above)
Comment (okawo80085) [reply index.html]: Lack of sleep is getting to me x)

Can you rephrase it?

File: index.html (hunk 2 above)
Comment (okawo80085) [reply index.html]: I should probably stop writing stuff when im 4 hours of sleep x)

File: index.html (hunk 1 above)
Comment (SimLeek) [reply index.html]: `Build a tracker, a controller, an HMD, or anything else, using whatever hardware your heart desires. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>. `

File: index.html (hunk 2 above)
Comment (SimLeek) [reply index.html]: For good measure:
`A good place to start is our <a href="https://github.com/HoboVR-Labs/issue_tracker/issues">issue tracker</a>, there you can find feature requests, bug reports, archives, conventions, and other issues.`

**Comment by SimLeek** [comment]
Looks good. 👍 
Merging...
